src/tactic/mk_opaque.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
+-/
+import tactic.core
+
+/-!
+# mk_opaque
+
+Tactic mk_opaque turns local definitions into local constants
+-/
+
+namespace tactic
+
+setup_tactic_parser
+
+/-- `mk_opaque x y z`, with `x`, `y`, `z` local definitions, transforms them into
+normal local constants
```

Comment:
Would it be appropriate to add a quick summary of why this tactic is useful here?